### PR TITLE
doc: clean legacy software SRAM names

### PR DESCRIPTION
**Pull request by yonghuah**
psram is legacy name of SSRAM, rename it to ssam

Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>

### CODE REVIEW

**Review by fitchbe — APPROVED** [review]

File: doc/user-guides/acrn-dm-parameters.rst [hunk 1]
```diff
@@ -462,9 +462,9 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
 ----
 
-``--psram``
+``--ssram``
    This option enables Pseudo (Software) SRAM passthrough to the VM.
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:466]: We could also update this free-text explanation to:
```suggestion
   This option enables Software SRAM passthrough to the VM.
```

**Review by gvancuts — CHANGES_REQUESTED** [review]
Can you fix the typo in your commit message; `ssam` -> `ssram`

There is another instance of `pSRAM` in `devicemodel/core/sw_load_common.c`, line 227. Can you fix it too?

**Comment by yonghuah** [comment]
@gvancuts  thank you for this catch, have updated this PR, pls review.

-Yonghua

**Review by ywan170 — APPROVED** [review]

**Review by gvancuts — APPROVED** [review]
Thanks!

**Pull request merged**